Library/Homebrew/cmd/desc.rb

```diff
@@ -0,0 +1,59 @@
+require 'descriptions'
+require 'cmd/search'
+
+USAGE =<<EOM
+Print formula descriptions.
+
+Usage:
+
+    brew desc --decache             Delete the cache, if present
+    brew desc --recache             [Re]generate the cache
+    brew desc name [name ...]       Describe the named formula(e)
+    brew desc (-s|-n|-d) <string>   Search specifed fields for string
+    brew desc (-s|-n|-d) /<regex>/  Search specifed fields for regex
+EOM
+
+
+module Homebrew
+  def desc
+    options = ARGV.options_only
+
+    if options.include?("--decache")
```

Comment:
It was meant to be either a portmanteau of “delete” and “cache”, or analogous to “decompress”. I’m open to clearer suggestions, though. `--delete_cache`?